src/physics/nonlinear_solid.cpp

```diff
@@ -227,8 +227,15 @@ void NonlinearSolid::InputInfo::defineInputFileSchema(axom::inlet::Table& table)
   traction_table.getField("y").defaultValue(1.0e-3);
   traction_table.getField("z").defaultValue(0.0);
 
-  auto& solver_table = table.addTable("solver", "Linear and Nonlinear Solver Parameters.");
-  serac::EquationSolver::defineInputFileSchema(solver_table);
+  auto& stiffness_solver_table =
+      table.addTable("stiffness_solver", "Linear and Nonlinear stiffness Solver Parameters.");
+  serac::EquationSolver::defineInputFileSchema(stiffness_solver_table);
+
+  if (dynamic) {
```

Comment:
Do we want to make this required based on `dynamic` instead of not defining the schema at all? If we toggle this way, it won't get written out to the documentation or if in the future we recreate a fully blown out input file.

Reply:
The problem is that Inlet's implementation of `contains` on a Table only checks if the child table or field exists, not whether it contains a value.  This is fixed in one of my Inlet PRs but for right now this appears to be the only way to check if an optional parameter was defined.